app/helpers/buyers/accounts_helper.rb

```diff
@@ -6,7 +6,7 @@ def public_domain(account)
   end
 
   def account_title account
-    [ h(account.org_name), h(account.admin_user.try!(:display_name)) ].compact.join(" &mdash; ").html_safe
+    [ h(account.org_name), h(account.decorate.admin_user_display_name) ].compact.join(" &mdash; ").html_safe
```

Comment:
what would happen what you would call decorate on a decorated object? nothing?

Reply:
It is not the case here, but I did it by accident in another place (fixed later), Gui spotted it, and it was working anyway XD: https://github.com/3scale/porta/pull/2024#discussion_r451387424

Look at this:
![image](https://user-images.githubusercontent.com/11318903/87309863-7708a480-c51d-11ea-8b3e-ba424af04622.png)


But anyway, it is not the case here, and I will move the `.decorate` to controllers instead of views/helpers in the following PR.